cmd/update.go

```diff
@@ -0,0 +1,197 @@
+package cmd
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/spf13/cobra"
+
+	"github.com/anchore/grype/internal/log"
+	"github.com/anchore/grype/internal/version"
+)
+
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "updates grype",
+	RunE:  runUpdate,
+}
+
+func init() {
+	rootCmd.AddCommand(updateCmd)
+}
+
+func runUpdate(_ *cobra.Command, _ []string) error {
+	fmt.Println("Checking for a newer version...")
+	newerVersionAvailable, desiredVersion, err := checkLatestVersion()
+	if err != nil {
+		panic(err)
```

Comment:
`panic` is rarely used as the correct error path in Golang code. It's only reserved for very niche cases where a program has no where else to go and needs to be extremely verbose about the error. 

Consider returning the error here